docker/build.sh

```diff
@@ -8,6 +8,7 @@ baseImageTable=(
     ["cuda12.1"]="nvidia/cuda:12.1.1-devel-ubuntu20.04"
     ["cuda12.2"]="nvidia/cuda:12.2.2-devel-ubuntu20.04"
     ["cuda12.3"]="nvidia/cuda:12.3.2-devel-ubuntu20.04"
+    ["rocm6.2"]="rocm/rocm-terminal:6.2"
```

Comment:
@Binyang2014 How about using another image that already has RCCL installed?

Reply:
AMD doesn't have much document about supported images. Only can find rocm/rocm-terminal, rocm/pytorch and rocm/tensorflow in the official docs, (https://rocm.docs.amd.com/projects/install-on-linux/en/latest/reference/docker-image-support-matrix.html) Let me ask AMD teams if they have dev image for this.